client/actions/fetchPostAction.js

```diff
@@ -11,16 +11,11 @@ const fetchPostSuccess = (payload) => {
 
 export const fetchPostRequest = (userData) => {
   return (dispatch) => {
-    debugger;
     const userId = localStorage.getItem('userId');
-    console.log(userData, userId, '==============');
     return axios.get(`/api/posts/${userData}/${userId}`).then((payload) => {
       if (payload.data.status === 'success') {
```

Comment:
Use status codes instead of messages (https://en.wikipedia.org/wiki/List_of_HTTP_status_codes)